app/views/content/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -78,7 +80,11 @@ You usually need the equivalent of a GCSE (grade 4) in English and maths.
 
 To teach children aged 3 to 11, you also need the equivalent of a GCSE (grade 4) in a science subject.
 
-[Contact UK ENIC to find out if your qualifications meet the standard](https://www.enic.org.uk/).
+#### To make sure your qualifications are equivalent
```

Comment:
```suggestion
#### Making sure your qualifications are equivalent
```

Reply:
This aims to follow on from an earlier header of 'You need:'